### ISSUE TITLE
Default openai url: "Error: OpenAPI health check failed"

### ISSUE THREAD

**Issue by CBenoit** [dustinblackman/oatmeal#12]
I’m configuring oatmeat via env variables in order to not leak my token through the CLI arguments (not sure if this is relevant for the issue at hand though).

For convenience, I have a script looking like this:
```sh
#!/bin/bash

export OATMEAL_BACKEND=openai
export OATMEAL_OPENAI_TOKEN=<redacted>
export OATMEAL_MODEL=gpt-4

oatmeal
```

The variables appear to be properly accounted for since I can see the following:
![image](https://github.com/dustinblackman/oatmeal/assets/3809077/935957b9-2a34-4681-9370-f31c23e05603)

However, I have no idea of why the health check is failing.

Accessing https://status.openai.com/api/v2/status.json is showing "All Systems Operational", and the error itself does not include much details.

Any way to extract additional information from oatmeal? I don’t see anything like `log` or `tracing` in the dependencies of the project, so I assume there is no way to make it write into some log file.

PS: cool project by the way.

**Comment by dustinblackman** [comment]
OpenAI broke the API index path which now 404s... that's nice. https://api.openai.com

I'll have to consider rewriting that to be better. 

As for logging, I have it on my TODOs to output to a temp file if enabled. :)

I'll get the patched up. Thanks for the report!

**Comment by CBenoit** [comment]
Oh I see, haha. Thank you! :pray:

**Comment by dustinblackman** [comment]
Fixed in [v0.7.0](https://github.com/dustinblackman/oatmeal/releases/tag/v0.7.0). Thanks!
